framework/include/outputs/OversampleOutput.h

```diff
@@ -82,11 +82,6 @@ class OversampleOutput :
    */
   virtual void updateOversample();
 
-  /**
-   * A pointer to the current mesh
-   * When using oversampling this points to the created oversampled, which must
-   * be cleaned up by the destructor.
-   */
   MooseMesh * _mesh_ptr;
```

Comment:
You removed all the documentation for this parameter... should the parameter itself be gone?  If not, it should probably have some doco.